src/resources/postcss/components/_events-bar.pcss

```diff
@@ -159,6 +159,40 @@
 		width: 21px;
 	}
 
+	.tribe-events-c-events-bar__tabs {
+		display: flex;
+	}
+
+	.tribe-events-c-events-bar__tab {
+		border-radius: var(--border-radius-default);
+		display: flex;
+		flex: 1 1 50%;
+		flex-direction: column;
+		align-items: center;
+		padding: var(--spacer-3) 0;
+
+		&:hover {
```

Comment:
also `&:focus`

Reply:
see note in fe-de-views chat on hover and active state

Reply:
see note in fe-de-views chat on hover and active state

Reply:
Ok, let's wait until they chime in for this one. My interpretation was that the hover had also the bg color.